ui/app/components/calendar-widget.js

```diff
@@ -42,7 +43,6 @@ class CalendarWidget extends Component {
   @tracked disablePastYear = this.isObsoleteYear(); // if obsolete year, disable left chevron
   @tracked disableFutureYear = this.isCurrentYear(); // if current year, disable right chevron
   @tracked showCalendar = false;
-  @tracked showSingleMonth = false;
```

Comment:
removed from calendar widget. This was a dropdown option we are no longer going to show in 1.10